scripts/effects/effect-definitions.js

```diff
@@ -3613,4 +3631,37 @@ export default class EffectDefinitions {
       ],
     });
   }
+
+  get _underwaterCombat() {
+    return this._effectHelpers.createActiveEffect({
+      name: 'Underwater Combat',
+      description: `- Disadvantage on melee weapon attacks, if a creature doesn't have a swimming speed (either natural or granted by magic), unless the weapon is a dagger, javelin, shortsword, spear, or trident.<br>- Disadvantage on ranged weapon attacks, unless the weapon is a crossbow, a net, or a weapon that is thrown like a javelin (including a spear, trident, or dart). <br>- A ranged weapon attack automatically misses a target beyond the weapon's normal range.<br>- Creatures have resistance to fire damage.`,
+      icon: 'icons/magic/water/wave-water-rolling-blue.webp',
```

Comment:
Could add `isViewable: myFunctionToSeeIfAllowed()`

Reply:
Yeah that would work!

I was thinking of submitting quite a few of these as I keep writing more midi specific ones.

It just occurred to me though, that it might be better to add them as a standalone daughter module, hooking on a relevant hook to add them in with the rest you provide 🤔

What do you think? Would that make sense?

Reply:
Here I am again, second guessing myself :)

It would only make sense to have a separate one for conditions or effects that are not translatable to WIRE compatible ones through the relevant flag.

But still even I am not convinced of the value of a separate module...

It might save you from some trouble keeping up with midi changes